cmd/ak/cmd/runtimes/run.go

```diff
@@ -125,14 +126,18 @@ func init() {
 	runCmd.Flags().BoolVar(&txtarFile, "txtar", false, "input file is a txtar archive containing program")
 	runCmd.Flags().BoolVarP(&emitValues, "values", "v", false, "emit result values")
 	runCmd.Flags().BoolVarP(&test, "test", "T", false, "fail if output is different than txtar comment. implies --txtar")
+	runCmd.Flags().BoolVarP(&quiet, "quiet", "q", false, "do not emit prints")
 }
 
 func run(ctx context.Context, b *sdkbuildfile.BuildFile, path string) (map[string]sdktypes.Value, []string, error) {
 	var prints []string
 
 	cbs := &sdkservices.RunCallbacks{
 		Print: func(_ context.Context, _ sdktypes.RunID, msg string) {
-			fmt.Println(msg)
+			if !quiet {
+				fmt.Println(msg)
```

Comment:
Why not log? (in human format)

Reply:
might append time and friends. we want to be able to compare it.